usagov/usagov-2021
# USAGOV-2047: Searchbox no longer a component, turn off sdc

In order to turn off the SDC module [before upgrading to 10.3] we just need to make our one-and-only component a non-component. 


https://cm-jira.usa.gov/browse/USAGOV-2047

## Description
Made the searchblock a non-component. And pushed the setting to uninstall the SDC module.

## Type of Changes


## Testing Instructions
Switch to this branch. Run `bin/drush cr` and then `bin/drush cim`. Confirm the SDC module is turned off. And to test; Go to the front-page (or any page) and confirm the page is *not* crashing [over trying to render the search-box].

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
